contracts/root/predicates/IPredicate.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity ^0.5.2;
+
+import { BytesLib } from "../../common/lib/BytesLib.sol";
+import { Common } from "../../common/lib/Common.sol";
+// import { MerklePatriciaProof } from "../../common/lib/MerklePatriciaProof.sol";
+import { RLPEncode } from "../../common/lib/RLPEncode.sol";
+import { RLPReader } from "solidity-rlp/contracts/RLPReader.sol";
+import { WithdrawManager } from "../withdrawManager/WithdrawManager.sol";
+
+contract IPredicate {
+  using RLPReader for RLPReader.RLPItem;
+  WithdrawManager internal withdrawManager;
+
+  constructor(address _withdrawManager) public {
+    withdrawManager = WithdrawManager(_withdrawManager);
+  }
+
+  function startExit(bytes memory data) public;
```

Comment:
Use interface `(address predicate, bytes memory data)` may be?

Reply:
We don't need `address predicate` because we are calling `startExit` directly in the predicate contract.